test/behaviour/mocks/default.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
```

Comment:
This file should really load the JS files from the api folder. By returning an array, it can specify multiple mocks. 

The module should iterate over the api folder, adding each JSON file as a mock for the appropiate path.